sky/engine/core/script/dart_init.cc

```diff
@@ -420,7 +430,7 @@ void InitDartVM() {
   }
   CHECK(Dart_SetVMFlags(args.size(), args.data()));
 
-  {
+  if (IsDevelopMode()) {
```

Comment:
The compiler was removing this dead code.  Changed this to an #ifdef for safety.